### PR TITLE
add save model algorithm

### PR DESCRIPTION
**Pull request by TomYOON**
you can save trained model  comfortably

### CODE REVIEW

**Comment by TomYOON** [comment]
auto nogada and save
please check the warning message in this code at training section
/usr/local/lib/python3.6/dist-packages/torch/nn/modules/loss.py:431: UserWarning: Using a target size (torch.Size([5, 1])) that is different to the input size (torch.Size([5])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
"UserWarning"

**Comment by TomYOON** [comment]
little modified

**Comment by TomYOON** [comment]
add prediction function using pretrained model

**Comment by TomYOON** [comment]
bug modify

**Comment by TomYOON** [comment]
성능 계산 추가
